source/js/tile-coverage-map.js

```diff
@@ -4,9 +4,7 @@ export default class TileCoverageMap
     {
         this._rows = rows;
         this._cols = cols;
-        // this._map = rows.fill(cols);
-        // this._map = fill(rows).map(() => fill(cols, false));
-        this._map = new Array(rows).fill(new Array(cols).fill(false));
+        this._map = new Array(rows).fill().map(() => new Array(cols).fill(false));
```

Comment:
No only the thick arrow and the warning is that it's only available for ES6, no other warnings. (From JSHint)